server/container/guice/protocols/webadmin-mailbox/src/main/java/org/apache/james/modules/server/RecomputeSingleComponentCurrentQuotasServicesExcludingJMAPModule.java

```diff
@@ -0,0 +1,35 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.modules.server;
+
+import org.apache.james.mailbox.quota.task.RecomputeMailboxCurrentQuotasService;
+import org.apache.james.mailbox.quota.task.RecomputeSingleComponentCurrentQuotasService;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.multibindings.Multibinder;
+
+public class RecomputeSingleComponentCurrentQuotasServicesExcludingJMAPModule extends AbstractModule {
```

Comment:
This too long of a class name is symptomatic: we have a problem here... 64 characters!

My bet is that it hould not exist.

My recommendation: 
 - Bind Mailbox recomputation service in `MailboxRoutesModule`
 - Bind Jmap recomputation service in `JmapTaskSerializationModule`
 - Bind Sieve recomputation service in `SieveRoutesModule`

That way no additional module is needed and the assembling would be automatically the good one.